bin/hive-config.sh

```diff
@@ -68,3 +68,7 @@ export HIVE_AUX_JARS_PATH=$HIVE_AUX_JARS_PATH
 
 # Default to use 256MB 
 export HADOOP_HEAPSIZE=${HADOOP_HEAPSIZE:-256}
+
+# Disable the JNDI. This feature has critical REC vulnerability.
+# when 2.x <= log4j.version <= 2.14.1
+export HADOOP_CLIENT_OPTS="$HADOOP_CLIENT_OPTS -Dlog4j2.formatMsgNoLookups=true"
```

Comment:
maybe we do not need this after updating?

Reply:
Similar HBase fix:
https://github.com/apache/hbase/pull/3933/files

The leader of knownsec 404 team (ZoomEye & SeeBug) 'Heige' have also recommend to set log4j2.formatMsgNoLookups to true
Ref: https://www.cyberkendra.com/2021/12/worst-log4j-rce-zeroday-dropped-on.html

Reply:
Got it, thank you!